src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -284,18 +285,25 @@ export class SignifyApi {
     aid: Aid,
     otherAids: Pick<Aid, "state">[],
     name: string,
-    delegate?: Aid
+    delegate?: Aid,
+    threshold?: number
   ): Promise<{
     op: any;
     icpResult: EventResult;
     name: string;
   }> {
+    if (!threshold) {
+      threshold = otherAids.length + 1;
+    }
```

Comment:
I think we can simplify this and just make `threshold` non optional, both here and in `identifierService.ts`. The UI has a component to select the threshold.